include/common_structs.h

```diff
@@ -1282,7 +1285,7 @@ typedef struct Actor {
     /* 0x211 */ u8 debuffDuration;
     /* 0x212 */ u8 staticStatus; /* 0B = yes */
     /* 0x213 */ u8 staticDuration;
-    /* 0x214 */ u8 stoneStatus; /* 0C = yes */
+    /* 0x214 */ s8 stoneStatus; /* 0C = yes */
     /* 0x215 */ u8 stoneDuration;
     /* 0x216 */ u8 koStatus; /* 0D = yes */
```

Comment:
Very likely all of these are s8 given that stoneStatus is s8.

Reply:
yeah, but we'll find out in due time when a function that uses them is decomped